.github/workflows/build.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       - name: Configure
         working-directory: ./build
         run: |
-          cmake .. ${{ matrix.config.cmake_extra_args }} -DCMAKE_BUILD_TYPE=${{ matrix.config.config }} -DSMALL_BUILD_WITH_PEDANTIC_WARNINGS=ON
+          cmake .. ${{ matrix.config.cmake_extra_args }} -DCMAKE_BUILD_TYPE=${{ matrix.config.config }} -DSMALL_BUILD_WITH_PEDANTIC_WARNINGS=ON  -DSMALL_DEVELOPER_MODE=ON
```

Comment:
`DSMALL_BUILD_WITH_PEDANTIC_WARNINGS` is already `ON` by default when `-DSMALL_DEVELOPER_MODE=ON`?

Reply:
Yes, you're right. I'll resolve it here.